js/router.js

```diff
@@ -80,47 +81,8 @@ class Router extends Backbone.Router {
   }
 
   handlePreview(...args) {
-    args = args.filter(v => v !== null);
```

Comment:
What was the purpose of all this code that is being deleted?

Reply:
All the deleted code is the same as what handleRoute already does, so I am just calling it to remove the duplication. The only exception being https://github.com/adaptlearning/adapt-contrib-core/pull/557/files#diff-2509a87ef563086fd634f97cc8372c3277e00655d6b9cbd7fc58c828ccdd0216R109-R111